### ISSUE TITLE
Livery Request: N57VB

### ISSUE THREAD

**Issue by tonghuix** [gilbertohasnofb/c172p-detailed-liveries#2]
http://airmart.com/aircraft-for-sale/178/cessna/n57vb-1985-cessna-172p-skyhawk/n57vb

I find this aircraft so beautiful, could any one made this livery?

![](http://airmart.com/sites/default/files/styles/slideshow-full/public/Right%20feature%201%20N57VB.jpg?itok=9ms2vj3e)

**Comment by gilbertohasnofb** [comment]
Yes, this livery is gorgeous. I will try to create it when I've got some free time.
